recipes/easchersim/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "easchersim" %}
+{% set version = "v1.2.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/c4341/easchersim/-/archive/v1.2.0/easchersim-v1.2.0.tar.gz
+  sha256: 863a1b297746572453f355b219a0f17d42a3cea8eb0b1b6e67219b39260948d3
+
+build:
+  noarch: python
+  number: 0
+  script: |
+    export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    {{ PYTHON }} -m pip install --no-deps . -vv
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools >=46
+    - setuptools_scm[toml] >=3.4
```

This syntax is Python specific, conda is Python agnostic.

```suggestion
    - setuptools_scm >=3.4
```